client/state/followers/actions.js

```diff
@@ -19,38 +10,82 @@ import {
 	FOLLOWER_REMOVE_SUCCESS,
 } from 'calypso/state/action-types';
 
-export default {
-	fetchFollowers( query, silentUpdate = false ) {
-		return ( dispatch ) => {
-			// TODO: Componentes should not fetch if already fetching
-			debug( 'fetching followers', query );
-			if ( ! silentUpdate ) {
-				dispatch( {
-					type: FOLLOWERS_REQUEST,
-					query,
-				} );
-			}
-			wpcom
-				.site( query.siteId )
-				.statsFollowers( query )
-				.then( ( data ) => dispatch( { type: FOLLOWERS_RECEIVE, query, data } ) )
-				.catch( ( error ) => dispatch( { type: FOLLOWERS_REQUEST_ERROR, query, error } ) );
-		};
-	},
-	removeFollower( siteId, follower ) {
-		return ( dispatch ) => {
-			debug( 'removing follower', follower, siteId );
-			dispatch( {
-				type: FOLLOWER_REMOVE_REQUEST,
-				siteId: siteId,
-				follower: follower,
-			} );
-			wpcom
-				.undocumented()
-				.site( siteId )
-				.removeFollower( follower.ID )
-				.then( ( data ) => dispatch( { type: FOLLOWER_REMOVE_SUCCESS, siteId, follower, data } ) )
-				.catch( ( error ) => dispatch( { type: FOLLOWER_REMOVE_ERROR, siteId, follower, error } ) );
-		};
-	},
-};
+/**
+ * Returns an action object used in signalling that followers for the site
+ * have been requested..
+ *
+ * @param {object} query params to be sent to the API
+ */
+export const requestFollowers = ( query ) => ( {
+	type: FOLLOWERS_REQUEST,
+	query,
+} );
+
+/**
+ * Returns an action object used in signalling that a request for followers
+ * was successfull.
+ *
+ * @param {object} query params which were sent to the API
+ * @param {object} data API response object
+ */
+export const successRequestFollowers = ( query, data ) => ( {
+	type: FOLLOWERS_RECEIVE,
+	query,
+	data,
+} );
+
+/**
+ * Returns an action object used in signalling that a request for followers
+ * has failed.
+ *
+ * @param {object} query params which were sent to the API
+ * @param {object} error contains the error response
+ */
+export const failRequestFollowers = ( query, error ) => ( {
+	type: FOLLOWERS_REQUEST_ERROR,
+	query,
+	error,
+} );
+
+/**
+ * Returns an action object used in signalling that a follower should
+ * be removed from the site.
+ *
+ * @param {number} siteId site identifier
+ * @param {object} follower follower object
```

Comment:
```
follower follower object
```

Seems like docs need some tweaking here and there. Sentence case for the description will fix the issue. This applies to multiple areas in this file.

Reply:
My editor doesn't complain about those. Does 8de8b7c work?

Reply:
Yep, that does the job for me. My editor didn't complain, either, but my eyes do 😬